Source/WebCore/Modules/mediastream/MediaTrackCapabilities.cpp

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(MEDIA_STREAM)
 
+#include "JSMeteringMode.h"
```

Comment:
Why do we need JSMeteringMode.h and not JSVideoFacingMode.h

Reply:
There is no JSVideoFacingMode.h because we don't define the FacingMode enum in idl. This patch was already big enough when I found that so I decided to do it in a followup.